packages/wallet/src/Wallets/BaseWallet.ts

```diff
@@ -727,25 +811,54 @@ export class PersonalWallet implements ObservableWallet {
     );
   }
 
-  async getPubDRepKey(): Promise<Ed25519PublicKeyHex> {
-    if (!this.drepPubKey) {
-      try {
-        this.drepPubKey = (await this.bip32Account.derivePublicKey(keyManagementUtil.DREP_KEY_DERIVATION_PATH)).hex();
-      } catch (error) {
-        this.#logger.error(error);
-        throw error;
-      }
+  async signData(props: SignDataProps): Promise<Cip30DataSignature> {
+    if (isBip32PublicCredentialsManager(this.#publicCredentialsManager)) {
+      return cip8.cip30signData({
+        // TODO: signData probably needs to be refactored out of the wallet and supported as a stand alone util
```

Comment:
Please capture in a ticket under Misc Technical Improvements epic
```suggestion
        // LW-XXXX: signData probably needs to be refactored out of the wallet and supported as a stand alone util
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1111/commits/8beeee61131bd339e57fe1dda59c6d85b3abd473 (ticket created)